docs/data/sql_functions.yml

```diff
@@ -643,6 +643,9 @@ collection:
   - sql: ARRAY_REVERSE(haystack)
     table: haystack.arrayReverse()
     description: Returns an array in reverse order. If the array itself is null, the function will return null.
+  - sql: ARRAY_INTERSECT(haystack, needle)
+    table: haystack.arrayIntersect(needle)
+    description: Returns an array that contains the common elements of both input arrays.
```

Comment:
it doesn't reflect the fact mentioned in PR's description about only one element in common.
E.g. `array1 = array[1, 1, 2, 2, 3, 3]` and `array2 = array[2, 3, 2, 3]`
what will the function return?

also probably need to change name of args (`needle` and `haystack`)